spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/channel/AbstractChannelMessageStoreQueryProvider.java

```diff
@@ -23,6 +23,7 @@
  *
  * @since 2.2
  */
+@Deprecated
 public abstract class AbstractChannelMessageStoreQueryProvider implements ChannelMessageStoreQueryProvider {
 
 	public String getCountAllMessagesInGroupQuery() {
```

Comment:
Since all methods have been moved to the interface, there is no need to keep them in this class any more.